sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -94,40 +96,101 @@ protected TimeSeriesInsightsClient()
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(GetModelSettings)}");
+            scope.Start();
+            try
+            {
+                Response<ModelSettingsResponse> modelSettings = await _modelSettingsRestClient.GetAsync(null, cancellationToken).ConfigureAwait(false);
+                TimeSeriesModelSettings timeSeriesModelSettings = modelSettings.Value.ModelSettings;
+                return Response.FromValue(timeSeriesModelSettings, modelSettings.GetRawResponse());
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
         }
 
         /// <summary>
-        /// Returns the model settings which includes model display name, Time Series ID properties and default type ID. Every Gen2 environment has a model that is automatically created.
+        /// Gets model settings synchronously.
         /// </summary>
-        /// <param name="clientSessionId"> Optional client session ID. Service records this value. Allows the service to trace a group of related operations across services, and allows the customer to contact support regarding a particular group of requests. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <returns>The deserialized application/json model settings digital twin and the http response <see cref="Response{ModelSettingsResponse}"/>.</returns>
-        public virtual Response<ModelSettingsResponse> Get(string clientSessionId = null, CancellationToken cancellationToken = default)
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The model settings which includes model display name, Time Series Id properties and default type Id with the http response <see cref="Response{T}"/>.</returns>
+        public virtual Response<TimeSeriesModelSettings> GetModelSettings(CancellationToken cancellationToken = default)
         {
-            return _modelSettingsRestClient.Get(clientSessionId, cancellationToken);
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(GetModelSettings)}");
+            scope.Start();
+            try
+            {
+                Response<ModelSettingsResponse> modelSettings = _modelSettingsRestClient.Get(null, cancellationToken);
+                TimeSeriesModelSettings timeSeriesModelSettings = modelSettings.Value.ModelSettings;
+                return Response.FromValue(timeSeriesModelSettings, modelSettings.GetRawResponse());
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
         }
 
         /// <summary>
-        /// Gets the scope for authentication/authorization policy.
+        /// Updates model settings, either the model name or default type Id asynchronously.
```

Comment:
Maybe reword to "Updates model settings model name or default type Id asynchronously."

Reply:
How about Updates model name or default type Id async...

Reply:
wording to "Updates model name or default type Id on Time Series model settings"